apps/browser/src/autofill/content/abstractions/autofill-init.ts

```diff
@@ -23,7 +24,8 @@ export type AutofillExtensionMessage = {
   data?: {
     direction?: "previous" | "next" | "current";
     forceCloseInlineMenu?: boolean;
-    inlineMenuVisibility?: number;
+    settingType?: CipherType;
+    newSettingValue?: InlineMenuVisibilitySetting | boolean;
```

Comment:
```suggestion
    inlineMenuVisibility?: InlineMenuVisibilitySetting;
```

Suggestion - Revert this

Reply:
4a355b2f085c0e2db21a4286a45218b192239fab